Source/gg2/Objects/InGameElements/Character.events/Create.xml

```diff
@@ -21,7 +21,7 @@
     canDoublejump = 0;
     canCloak = 0;
     canBuild = 0;
-    jumpStrength = 8-(0.6/2); // reduce the part of the initial frame of gravity that we don't apply anymore (not framerate dependent, meant to match old 30fps arc)
+    jumpStrength = 8+(0.6/2); // reduce the part of the initial frame of gravity that we don't apply anymore (not framerate dependent, meant to match old 30fps arc)
```

Comment:
"gg2 used to not apply gravity on the first frame after jumping. Now it does. This means the initial velocity of a jump has to be stronger by the difference. (value change not framerate depenent; also meant to canonical jump arc)"